### PR TITLE
CE-2784 Add basic Template Classification entry point to edit page

### PR DESCRIPTION
**Pull request by adamkarminski**
This PR makes use of EditPage's hook `EditPageLayoutExecute` to inject HTML with the Template Classification edit entry point. I've had to make some moves around the mocked backend as well to simplify using it from both JS and PHP.

/cc: @Wikia/community-engineering @kamilkoterba

### CODE REVIEW

File: extensions/wikia/TemplateClassification/TemplateClassificationView.php [hunk 1]
```diff
@@ -5,44 +5,48 @@
...
+		$templateType = $this->renderTemplateType( $articleId, $user );
+		return \MustacheService::getInstance()->render(
+			__DIR__ . '/templates/TemplateClassificationEditPageEntryPoint.mustache',
 			[
-				'articleId' => $articleId,
-			],
-			true, \WikiaRequest::EXCEPTION_MODE_RETURN
+				'header' => wfMessage( 'template-classification-type-header' ),
+				'templateType' => $templateType,
+			]
```
Comment (kamilkoterba) [inline extensions/wikia/TemplateClassification/TemplateClassificationView.php:48]: Use a partial to render the entry point itself.

File: extensions/wikia/EditPageLayout/templates/EditPageLayout_EditPage.php [hunk 2]
```diff
@@ -124,6 +124,7 @@
 									</a>
 								</div>
 							<?php endif ?>
+							<?php if ( $extraPageControlsHtml ) print $extraPageControlsHtml ?>
```
Comment (kamilkoterba) [inline extensions/wikia/EditPageLayout/templates/EditPageLayout_EditPage.php]: Wrap in brackets or : ... endif;

**Comment by kamilkoterba** [comment]
LGTM

File: extensions/wikia/TemplateClassification/TemplateClassificationView.php (hunk 1 above)
Comment (adamkarminski) [inline extensions/wikia/TemplateClassification/TemplateClassificationView.php:48]: Would have to refactor the PHP code to a mustache template - later.

**Pull request merged**